R/preprocess.R

```diff
@@ -404,8 +404,7 @@ enw_filter_report_dates <- function(obs, latest_date, remove_days) {
 #'
 #' @param include_days if \code{earilest_date} is not given, the number
 #' of reference dates to include, ending with the latest reference
-#' date included once report dates have been removed. If specified
-#' this is indexed to `latest_date` or `remove_days`.
+#' date included (determined by `latest_date` or `remove_days`).
```

Comment:
I am not 100% sure what "once report dates have been removed" meant, but I think it was wrong. The report dates smaller than the earliest date were only removed in the end.